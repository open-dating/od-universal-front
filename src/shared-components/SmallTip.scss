.small-tip {
  margin: 0;

  > div {
    width: 100%;
  }

  small {
    color: rgba(0, 0, 0, 0.54);
  }
}
