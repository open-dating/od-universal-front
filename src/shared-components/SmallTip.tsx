import React, {ReactElement} from 'react'

import './SmallTip.scss'

export function SmallTip({children}: {children: ReactElement}) {
  return (
    <div className="small-tip">
      <small>
        {children}
      </small>
    </div>
  )
}
