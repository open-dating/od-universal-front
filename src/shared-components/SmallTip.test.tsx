import React from 'react'
import {shallow} from 'enzyme'
import {SmallTip} from './SmallTip'

describe('SmallTip', () => {
  it('toMatchSnapshot', () => {
    const wrapper = shallow(<SmallTip>
      <div>Some tip text</div>
    </SmallTip>)

    expect(wrapper).toMatchSnapshot()
  })
})
