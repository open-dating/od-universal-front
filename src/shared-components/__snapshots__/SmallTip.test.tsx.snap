// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallTip toMatchSnapshot 1`] = `
<div
  className="small-tip"
>
  <small>
    <div>
      Some tip text
    </div>
  </small>
</div>
`;
